lib/middleware/errors.js

```diff
@@ -19,19 +20,25 @@ function _applicationError(err, req, res, next) {
   logger.log('error', {'Error stack': err.stack});
 
   next(err);
+
+  return Promise.resolve();
```

Comment:
Don't think this line would be reached (or useful) as `next()` sends the router to the next bit of middleware.